iterators.tex

```diff
@@ -623,31 +650,186 @@
 \textit{p}).
 \exitexample
 
-\indexlibrary{\idxcode{Iterator}}%
-\begin{codeblock}
+{\color{newclr}
+\rSec2[iterators.sizediterator]{Concept SizedIterator}
+\pnum
+The \tcode{SizedIterator} concept refines \tcode{Iterator} with the
+capability to determine the distance between two iterators that denote
+elements of the same range in constant time with the \tcode{-} operator.
+
+\indexlibrary{\idxcode{SizedIterator}}%
+\begin{itemdecl}
+  template <class>
+  constexpr bool disable_sized = false;
```

Comment:
The concepts `uncvref` the type before checking `disable_sized`. Explicitly in `SizedRange`, and implicitly in `SizedIterator` in that the library functions take iterators by value so the type comes pre-decayed. I should make that fact explicit in `SizedIterator` as well for clarity. I find it unlikely there's an iterator/range type out there whose size-ability depends on cv-qualification; it's not worth the trouble to try to support such a creature.

Regarding the sentinel type and `disable_sized`: sentinels are new. We don't need to support broken underconstrained sentinels as we do iterators since there are none in the wild.